### PR TITLE
Add APLv2 license

### PR DESCRIPTION
**Pull request by Koslun**
Adding Apache License per our agreement. Copied from https://github.com/moby/moby/blob/master/LICENSE

Slack message confirmation from VP of engineering: https://getnormative.slack.com/archives/C0117F5JG8M/p1632928546200900?thread_ts=1632920668.189700&cid=C0117F5JG8M

### CODE REVIEW

**Comment by twentyrogersc** [comment]
Did you know you can

**Review by Hibbins — APPROVED** [review]
LGTM!

**Review by twentyrogersc — APPROVED** [review]
LGTM.

Unsure whether it's relevant, but instead of copying you can do it through the UI (and I think it adds badges/extra UI to the repos too). But it might detect it from this anyway... 🤔

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2013-2018 Docker, Inc.
```
Comment (twentyrogersc) [inline LICENSE:179]: Actually, might be worth using the Github template as this shouldn't be Docker (guessing Normative)?

**Review by twentyrogersc — CHANGES_REQUESTED** [review]

**Comment by Koslun** [comment]
@twentyrogersc good suggestion to use the template. Created a commit using it and put it in this PR instead: https://github.com/normative-io/normative-ngx-modules/pull/3

**Pull request closed without merging**